charts/warpstream-agent/templates/configmap.yaml

```diff
@@ -8,8 +8,18 @@ metadata:
 data:
   # The config values below are not required in playground mode.
   {{- if not .Values.config.playground }}
+  {{- if .Values.config.ingestionBucketURL }}
```

Comment:
do you need the inverse of this when compactionBucketURL is set but not ingestionBucketURL?